CMakeLists.txt

```diff
@@ -62,6 +62,15 @@ option(SIRF_INSTALL_DEPENDENCIES "Install dlls etc" WIN32)
 ####### CMake path
 set (CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
+# Define the pre-commit hook for developer
+find_package(Git)
+if(GIT_FOUND)
+  message(STATUS "Found Git")
+  file(COPY "${CMAKE_SOURCE_DIR}/update_copyright_hook" DESTINATION "${CMAKE_SOURCE_DIR}/.git/hooks" USE_SOURCE_PERMISSIONS)
```

Comment:
would this stuff be run at every `cmake`?

Reply:
also, I have a slight preference for calling the file `update_copyright_hook.sh`. It doesn't matter, but makes it maybe a bit clearer what the file is

Reply:
> would this stuff be run at every cmake?

Yes, could put it in an `if` statement to only run first time. But then what if we wanted to change the script on the master branch. We'd want everyone that pulled to get the new version. Doesn't seem too harmful to run each time.